pyproject.toml

```diff
@@ -23,7 +23,7 @@ repository = "https://github.com/z3z1ma/dbt-osmosis"
 [tool.poetry.dependencies]
 python = ">=3.8,<3.9.7 || >3.9.7,<3.12"
 click = ">7"
-dbt-core = ">=1"
+dbt-core = ">=1.6,<1.9"
```

Comment:
To be precise, [dbt-core 1.8 currently does not work with dbt-osmosis](https://github.com/z3z1ma/dbt-osmosis/issues/139), but I think it will be supported soon by [this pull request](https://github.com/z3z1ma/dbt-osmosis/pull/149), so I wrote it this way.